zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/CollectionDetails.java

```diff
@@ -109,16 +108,9 @@ public CollectionDetail get(HttpServletRequest request, HttpServletResponse resp
         addEventsForDetails(result.complete, collection);
         addEventsForDetails(result.reviewed, collection);
 
-        Set<Integer> teamIds = zebedeeCmsService.getPermissions().listViewerTeams(collection.description,
-                session);
-        // TODO: Remove feature flag after migration to dp-identity-api
-        if (cmsFeatureFlags().isJwtSessionsEnabled()) {
-            // TODO: remove the conversion from Integer to String after the permissions have been updated to be a string
-            result.setTeams(teamIds.stream().map(id -> Integer.toString(id)).collect(Collectors.toList()));
-        } else {
-            result.teamsDetails = zebedeeCmsService.getZebedee().getTeamsService().resolveTeamDetails(teamIds);
-            result.teamsDetails.forEach(team -> collection.getDescription().getTeams().add(team.getName()));
-        }
```

Comment:
This logic is less than ideal and has been removed in favour of a more in depth rework of the viewer team logic.